README.md

```diff
@@ -12,24 +12,18 @@ Refer to `model_results.csv` for model versions and their metrics.
 
 ### HuggingFace Hub
 
-In order to pull the model from HuggingFace, you need to be logged in:
-
-```Bash
-huggingface-cli login
-```
-
-Then you can pull the model from a Python script:
+Pull the RADIO model from a Python script:
 
 ```Python
 from transformers import AutoModel
 model = AutoModel.from_pretrained("nvidia/RADIO", trust_remote_code=True)
 ```
 
-Alternatively, you can specify an access token:
+Pull the E-RADIO model from a Python script:
 
 ```Python
-access_token = "<YOUR ACCESS TOKEN"
-model = AutoModel.from_pretrained("nvidia/RADIO", trust_remote_code=True, token=access_token)
```

Comment:
Would we want to keep both versions? Or is it configured somewhere else?

Reply:
Hi, we don't need an access token if the model is not gated.

Reply:
Oh, you're right. I misread the diff, and thought that you also changed "RADIO" to "E-RADIO". I see it above now.